{% extends "base.html" %}

{% load static %}

    {% block head_title %}About{% endblock head_title %}

      {% block nav %}
      <li role="presentation" class="active"><a href="{% url 'about' %}">About</a></li>
      <li role="presentation"><a href="{% url 'access' %}">Access</a></li>
      <li role="presentation"><a href="{% url 'submit' %}">Submit</a></li>
      <li role="presentation"><a href="{% url 'news' %}">News</a></li>
      {% endblock nav %}

    {% block container %}
    <div class="row" style="margin-top: 10px; margin-bottom: 100px;">
      <div class="col-md-2 col-sm-2 col-xs-2">
        <ul class="nav nav-pills nav-stacked">
          <li role="presentation" class="active"><a href="#about_trpkb" data-toggle="tab">About RadioGKB</a></li>
          <li role="presentation"><a href="#cite" data-toggle="tab">Cite</a></li>
          <li role="presentation"><a href="#contact" data-toggle="tab">Contact</a></li>
          <li role="presentation"><a href="#contribution" data-toggle="tab">Contribution</a></li>
        </ul>
      </div>
      <div class="col-md-10 col-sm-10 col-xs-10">
        <div class="tab-content">
          <div id="about_trpkb" class="tab-pane fade in active">
            <div class="panel panel-default">
              <div class="panel-body">
                <h1>About Knowledgebase</h1>
                <p>The RadioGKB (Radiation Genomics Knowledgebase) is a tumor knowledge resource that focus on the relationship between human genetic variation or gene expression and prognosis of patients who received radiotherapy with or without other treatments. RadioGKB collects, curates and disseminates knowledge about the impact of human genetic variation on prognosis of cancer patients. Now, we mainly focus on the association between single nucleotide polymorphism (SNP) or gene expression and prognosis. Mutations in other type will be included in turn.</p>
                <br>
                <div class="img-thumbnail text-center">
                  <img style="max-width:100%;" src="/static/images/about_RadioGKB_Structure.png" alt="Figure 1 The RadioGKB Knowledge Structure">
                  <strong>Figure 1. The RadioGKB Knowledge Structure</strong>
                </div>
                <br>
                <br>
                <p><strong>Figure 1 The RadioGKB Knowledge Structure</strong> Providing users with a visualization of the different types of information found in knowledgebase, and how this knowledge is acquired and integrated together – from the data of variation and prognosis at the outside of the circle to implementation of genetic variation in the clinic at the central. Each step is described briefly below.</p>
                <h3 style="color:#6699FF;">Tumor Prognosis Literature</h3>
                <p>The tumor prognosis literature, such as publications found in PubMed and published by researchers and clinicians forms the outside of the RadioGKB knowledge structure. From here, we include literature according to inclusion criteria. For every study included in the RadioGKB, we classify them by Evidence-based Medicine Level from iCMDB knowledgebase (<a href="http://www.vishuo.com/cn/icmdb/">http://www.vishuo.com/cn/icmdb/</a>).</p>
                <br>
                <div class="img-thumbnail text-center">
                  <img style="max-width:100%;" src="/static/images/about_EBML.png" alt="Figure 2 Evidence-based Medicine Level">
                  <strong>Figure 2. Evidence-based Medicine Level</strong>
                </div>
                <br>
                <br>
                <h3 style="color:#6699FF;">Knowledge Extraction</h3>
                <p>We extract some clinical information, such as patient number, sex ratio, age, genetic variation, treatment, prognosis and statistical results from the publication.</p>
                <h3 style="color:#6699FF;">Knowledge Annotation, Aggregation & Integration</h3>
                <p>We replace name of gene and variation with official name. At the same time, we collect the primary information of gene and variation. Then we summary up the treatment in clinical terms.</p>
                <h3 style="color:#6699FF;">Clinical Implementation</h3>
                <p>Base on the statistical result, we would know whether the genetic variation or gene expression could influence the prognosis with specific treatment.</p>
                <h1>About Knowledge</h1>
                <p>We focus on the association between single nucleotide polymorphism (SNP) and prognosis of cancer patients who received radiotherapy with or without other treatments. As we know, most SNP in human genome has two alleles: ancestral allele and minor allele. So, there are three genotypes in this specific position: wild homozygous type, mutant heterozygous type and mutant homozygous type.</p>
                <p>In clinical cohort study, we need to know whether exposure group influences the study endpoint when compared with un-exposure group under specific circumstance. In our knowledgebase, the exposure group often means wild homozygous type, and the un-exposure group often means mutant heterozygous type and mutant homozygous type or mutant heterozygous/homozygous type. For endpoint, it stands for specific event of prognosis. For example, most research use death as the endpoint in overall survival (OS).</p>
                <p>OR (odds ratio), HR (hazard ratio) and RR (relative risk/risk ratio) are extracted from publications to show the association between SNP and prognosis. Some articles do not offer OR sometimes, OR need to be calculated by using STATA 14.1 (StataCorp. 2015. Stata Statistical Software: Release 14. College Station, TX: StataCorp LP.) with original data.</p>
                <br>
                <div class="row">
                  <div class="col-md-1 col-sm-1 col-xs-1">
                  </div>
                  <div class="col-md-10 col-sm-10 col-xs-10">
                    <div class="text-center">
                      <strong>Table 1. Four-fold table</strong>
                    </div>
                    <table class="table-threeline">
                      <thead>
                        <tr>
                          <th>Group</th>
                          <th>Event</th>
                          <th>Non-event</th>
                          <th>Total</th>
                        </tr>
                      </thead>
                      <tbody>
                        <tr>
                          <td>Exposure</td>
                          <td><span style="color:red;">a</span></td>
                          <td><span style="color:red;">b</span></td>
                          <td>a+b</td>
                        </tr>
                        <tr>
                          <td>Un-exposure</td>
                          <td><span style="color:red;">c</span></td>
                          <td><span style="color:red;">d</span></td>
                          <td>c+d</td>
                        </tr>
                        <tr>
                          <td>Total</td>
                          <td>a+c</td>
                          <td>b+d</td>
                          <td></td>
                        </tr>
                      </tbody>
                    </table>
                  </div>
                  <div class="col-md-1 col-sm-1 col-xs-1">
                  </div>
                </div>
                <div class="text-center">
                  <img style="max-width:100%;" src="/static/images/about_formula.png">
                </div>
                <br>
                <br>
                <p>OR often be used in case-control study or prospective study. HR often been used clinical medicine research or cohort study of Epidemiology. Hazard is an instantons rate of a given outcome, which include time factor. HR is a ratio of two hazards (Exposure/Un-exposure) at that given time. RR often be used in cohort study or randomized controlled trial.</p>
                <p>If both OR/HR/RR and low value of 95% confidence interval (95% CI) > 1, the patients in exposure group has high risk to reach the endpoint compared with un-exposure group. On the contrary, the patients in exposure group has low risk to reach the endpoint. To be more accurate, we consider P value > 0.05 as significance level.</p>
                <p>In the result page, we use table to present the association between SNP and prognosis. Table 2 shows the association between XRCC1 rs25487 and ≥2 radiation pneumonitis in patients after treatment for non-small cell lung cancer.</p>
                <br>
                <div class="row">
                  <div class="col-md-1 col-sm-1 col-xs-1">
                  </div>
                  <div class="col-md-10 col-sm-10 col-xs-10">
                    <div class="text-center">
                      <strong>Table 2. The association between XRCC1 rs25487 and ≥2 radiation pneumonitis in patients after treatment for non-small cell lung cancer</strong>
                    </div>
                    <table class="table-threeline">
                      <thead>
                        <tr>
                          <th>Genotypes</th>
                          <th>No. of patients</th>
                          <th>Event</th>
                          <th>HR (U)</th>
                          <th>95%CI (U)</th>
                          <th>P (U)</th>
                          <th>HR (M)</th>
                          <th>95%CI (M)</th>
                          <th>P (M)</th>
                        </tr>
                      </thead>
                      <tbody>
                        <tr>
                          <td>GG</td>
                          <td>49</td>
                          <td>25</td>
                          <td>1.00</td>
                          <td></td>
                          <td></td>
                          <td>1.00</td>
                          <td></td>
                          <td></td>
                        </tr>
                        <tr>
                          <td>AG</td>
                          <td>72</td>
                          <td>28</td>
                          <td>0.73</td>
                          <td>0.42-1.25</td>
                          <td>0.248</td>
                          <td>0.76</td>
                          <td>0.44-1.31</td>
                          <td>0.322</td>
                        </tr>
                        <tr>
                          <td>AA</td>
                          <td>44</td>
                          <td>13</td>
                          <td>0.52</td>
                          <td>0.26-1.01</td>
                          <td>0.054</td>
                          <td>0.48</td>
                          <td>0.24-0.97</td>
                          <td>0.041</td>
                        </tr>
                        <tr>
                          <td>AG/AA</td>
                          <td>116</td>
                          <td>41</td>
                          <td>0.64</td>
                          <td>0.39-1.06</td>
                          <td>0.081</td>
                          <td>0.65</td>
                          <td>0.39-1.09</td>
                          <td>0.100</td>
                        </tr>
                      </tbody>
                    </table>
                  </div>
                  <div class="col-md-1 col-sm-1 col-xs-1">
                  </div>
                </div>
                <p><i>HR: hazard ratio; 95% CI: 95% confidence interval; U: univariate analysis; M: multivariate analysis, it means the HR was adjusted for some clinical factors. Event: ≥2 radiation pneumonitis.</i></p>
                <p>The data in table 2 comes from the article below.</p>
                <p><i>Yin M, Liao Z, Liu Z, et al. Functional polymorphisms of base excision repair genes XRCC1 and APEX1 predict risk of radiation pneumonitis in patients with non–small cell lung cancer treated with definitive radiation therapy[J]. International Journal of Radiation Oncology* Biology* Physics, 2011, 81(3): e67-e73. </i></p>
              </div>
            </div>
          </div>

          <div id="cite" class="tab-pane fade">
            <div class="panel panel-default">
              <div class="panel-body">
                <p>Coming soon!</p>
                <br><br><br><br><br><br><br><br><br>
              </div>
            </div>
          </div>

          <div id="contact" class="tab-pane fade">
            <div class="panel panel-default">
              <div class="panel-body">
                <p>Coming soon!</p>
                <br><br><br><br><br><br><br><br><br>
              </div>
            </div>
          </div>

          <div id="contribution" class="tab-pane fade">
            <div class="panel panel-default">
              <div class="panel-body">
                <p>Coming soon!</p>
                <br><br><br><br><br><br><br><br><br>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    {% endblock container %}
